finance_dl/schwab.py

```diff
@@ -206,36 +206,35 @@ def download_lot_details(self, pos_dir: str) -> None:
 
         logger.info("Getting lot details.")
 
-        lot_rows = self.get_elements_wait("table.securityTable tr.data-row")
+        lot_rows = self.get_elements_wait("table.securityTable tr.data-shrinkedrow")
```

Comment:
So I don't know why we are served differing HTML, but I tried just now and this change does not work for me. I get a timeout looking for these elements, they still have class `data-row` for me.

So it seems we will need to adapt this to not depend on that class. Perhaps it is enough to just look for `table.securityTable tr` and then rely on other checks below to filter out any non-security rows we happen to match?

Reply:
`table.securityTable tr` works for me as well. Updated the PR.